admin_manual/configuration/server/security/ransomware-protection.rst

```diff
@@ -16,32 +16,64 @@ About Ransomware Protection
 ---------------------------
 
 The app is tasked with *detecting*, *preventing*, and *reverting* anomalies.
-Anomalies are file operations (including *create*, *update*, *delete*, and *move*) that are not intentionally conducted by the user.
-It aims to do so in two ways: `prevention <ransomeware_prevention_label>`_, and `protection <ransomeware_protection_label>`_.
+Anomalies are file operations (including *create*, *update*, *delete*, and *move*) not intentionally conducted by the user.
+It aims to do so in two ways: `prevention <ransomware_prevention_label>`_, and `protection <ransomware_protection_label>`_.
 
-.. _ransomeware_prevention_label:
+.. _ransomware_prevention_label:
 
 Prevention: Blocking Common Ransomware File Extensions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Like other forms of cyberattack, Ransomware has a range of diverse characteristics.
-On the one hand it makes them hard to detect and on the other makes them even harder to prevent.
-Recent ransomeware attacks either encrypt a user's files and add a certain file extension to them (e.g., ".crypt") or they replace the original files with an encrypted copy and a certain file extension.
+Like other forms of cyberattack, ransomware has a range of diverse characteristics.
+On the one hand it makes them hard to detect, and on the other it makes them even harder to prevent.
+Recent ransomware attacks either encrypt a user's files and add a specific file extension to them (e.g., ".crypt"), or they replace the original files with an encrypted copy and add a particular file extension.
+
+File Extension Blacklist
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first line of defense against such threats is a blacklist that blocks write access to file extensions known to originate from ransomware.
+
+.. note:: 
+   Write access (e.g., moving and deleting files) is still possible for users, when they log in with their web browser.
```

Comment:
I am confused. This note belongs to the "Account locking" section. Trying to explain for the sake of clarity.
There are three elements of the prevention part:
1) Blacklist: A list of file name patterns known to originate from ransomware
2) Blocking (always enabled): On file upload a matching with the blacklist happens. When match is found, upload is denied. That's it.
3) Locking (optional): When the blocker finds a match the WebDAV API is set to read-only for the respective account making it impossible for clients to upload/move/delete etc. anything further.

Reply:
I might have moved that note around by mistake. Thanks for pointing it out.